gdc_client/download/client.py

```diff
@@ -136,7 +141,7 @@ def _post(self, path, headers={}, json={}, stream=True):
                 json=json,
                 headers=headers,
             )
-            if r.status_code != requests.codes.ok:
+            if r.status_code not in [requests.codes.ok, 203]:
```

Comment:
If you want, for consistency, you can change `requests.codes.ok` to `200`. Pylint also complains about it because of how requests is set up, so I don't feel bad about changing it. :p

Reply:
ok 😄